LibCarla/source/carla/NonCopyable.h

```diff
@@ -8,15 +8,30 @@
 
 namespace carla {
 
-  /// Inherit (privately) to suppress copy-construction and copy-assignment.
+  /// Inherit (privately) to suppress copy/move construction and assignment.
```

Comment:
Why not just use https://www.boost.org/doc/libs/1_63_0/libs/core/doc/html/core/noncopyable.html ?

Reply:
Yeah, this comes back from the time we didn't include boost. I just kept using it for consistency, we could alias it here.